### DISCUSSION TITLE
How do you customize the lualine extensions

### DISCUSSION THREAD

**Discussion by mandaputtra** [LunarVim/LunarVim#1923] [Q&A] [answered]
Sometimes I encounter the theme that I like doesn't have a match with the `whichkey` background, and statusline backgroun. How do I change the statusline and whichkey bacground on `config.lua`

![Screenshot from 2021-11-08 01-38-35](https://user-images.githubusercontent.com/23342943/140657586-ff43986a-f003-4613-b1d6-4185812e66b6.png)

**Comment by LostInTheLogs** [comment, marked as answer]
https://www.lunarvim.org/docs/master/configuration/appearance/colorschemes#customizing-some-colors
